extensions/noteplan-3/src/lib/note-utilities.ts

```diff
@@ -25,8 +30,16 @@ export interface Note {
   content: string;
 }
 
+const getPreferences = () => {
+  return getPreferenceValues<Preferences>();
+};
+
+export const notesExtension = () => {
+  return getPreferences().extension;
+};
+
 export const listNotes = (): NoteEntry[] => {
-  const paths = find(`${NOTE_PLAN_URI}/**/*.txt`, { absolute: true });
+  const paths = find(`${NOTE_PLAN_URI}/Notes/**/*.${notesExtension()}`, { absolute: true });
```

Comment:
Why have you changed the path by adding `/Notes/` here? I'm not familiar with the noteplan but wouldn't it break some cases?

```suggestion
  const paths = find(`${NOTE_PLAN_URI}/Notes/**/*.${preferences.extension}`, { absolute: true });
```

Reply:
To be honest I'm surprised it worked before; I assume the original author doesn't have any backups (these are triggered manually) to mess things up. NotePlan's app support directory structure:

<img width="489" alt="CleanShot 2022-08-01 at 15 16 26@2x" src="https://user-images.githubusercontent.com/878/182167988-f5fbc1da-f6a8-43cc-b28b-2d8bc1399c7e.png">